src/Api.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+namespace atk4\api;
+
+/**
+ * Main API class.
+ */
+class Api
+{
+    // @var \Zend\Diactoros\ServerRequest Request object
+    public $request;
```

Comment:
More likely these two (request and response) can be set as protected too, but ... maybe they can be used in some way outside of the class too for debug for example.

Reply:
i'm using them publicly though.